common/pref_names.cc

```diff
@@ -62,6 +62,8 @@ const char kNewTabPageShowStats[] = "brave.new_tab_page.show_stats";
 const char kNewTabPageShowRewards[] = "brave.new_tab_page.show_rewards";
 const char kNewTabPageShowBinance[] = "brave.new_tab_page.show_binance";
 const char kNewTabPageShowGemini[] = "brave.new_tab_page.show_gemini";
+const char kNewTabPageShowBitcoinDotCom[] =
+    "brave.new_tab_page.show_bitcoin_dot_com";
```

Comment:
This should go in `components/moonpay/common/moonpay_pref_names.cc` instead

Reply:
moved